pootle/apps/pootle_format/default.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+
+POOTLE_FORMATS = [
+    ("po",
+     dict(title='Gettext PO file',
+          extension="po",
+          template_extension="pot")),
+    ("xliff",
+     dict(title='XLIFF Translation File',
+          extension="xliff",
+          template_extension="xliff")),
+    ("xlf",
+     dict(title='XLIFF Translation File',
```

Comment:
How are you going to differentiate both XLIFF entries if they have the same title?